### PR TITLE
Add a new statement in home page

### LINKED ISSUE #466: Add a new statement in home page
**Is your feature request related to a problem? Please describe.**
In the first page, there should be a statement in between our flagship programs and the Student/Expert/Volunteer section saying " You could be a student, expert or a volunteer. At SEF have something in store for everyone "

### PR DESCRIPTION
**Pull request by Suthan97**
## Purpose
Added a new statement on home page
The purpose of this PR is to fix #466 

## Goals
A statement will display between our flagship programs and the Student/Expert/Volunteer section.
## Approach

- created a class section (new-statement)

- used text-centre

- used bold (`<b>`)

- typed statement text

### Screenshots
image

  
### Preview Link

https://pr-477-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by jayasanka-sack** [comment]
Good work @Suthan97 ! 
By the way, the title should be a little bit bigger.  Also, the PR link should be https://pr-477-sef-site.surge.sh/ (The PR number is 477)

You are doing great! Keep the good work up!

**Comment by jayasanka-sack** [comment]
Btw, as per the discussion on #466 another one is already working on this. Could you please ask him if he is still working on the issue? (You can add a comment on issue #466 and ask from him) 
Cheers! 🍻

**Comment by Suthan97** [comment]
I think he is already working on it, so he will do it better. 😊

**Comment by Suthan97** [comment]
Thank you so much. I will do future work completely correct way. 👍

**Comment by osusara** [comment]
@Suthan97 Nice work brother. I unassigned my self from this issue. You can do it.

**Comment by osusara** [comment]
@Suthan97 It's better if, you use <strong> rather than <b>. Also, the title font-size should be little bigger and bring it to the middle between our flagship programs and the Student/Expert/Volunteer sections.

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -23,286 +23,260 @@
...
-                                        <li>Ability to make an impact on the Sri Lankan education system.</li>
-                                        <li>Chance to connect with young talent in Sri Lanka.</li>
-                                    </ul>
-                                </div>
-                                <div class="tab-pane fade px-5 py-3" id="tabs-text-3" role="tabpanel"
-                                     aria-labelledby="tabs-text-3-tab">
-                                    <ul class="description text-justify text-dark font-weight-400">
-                                        <li>Develop In demand, skills needed for the job market when you graduate.</li>
+        </section>
+        <section class="new-statement text-center">
```
Comment (Gravewalker666) [inline src/main/webapp/index.html]: @Suthan97 Can you please contain this statement in a container

**Comment by Suthan97** [comment]
Ok I will do it 👍

File: src/main/webapp/index.html (hunk 1 above)
Comment (Suthan97) [reply src/main/webapp/index.html]: I did it. Please check 😊. I apologise for the delay.😔

**Comment by Suthan97** [comment]
> @Suthan97
> There you go, buddy! Just what the doctor ordered.
> The new statement looks great but you have messed up the code above it.
> Can you fix it so that we can merge this PR?
> 
> * This is the issue I'm talking about
>   image

Ok I will fix it.

**Pull request closed without merging**